dist/profile/templates/confluence/vhost.conf

```diff
@@ -2853,12 +2853,84 @@ RewriteRule "^/display/JENKINS/Subversion\+Plugin$" "https://plugins.jenkins.io/
+RewriteRule "^/display/JENKINS/Git\+Changelog\+Plugin$" "https://plugins.jenkins.io/git-changelog" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Robot\+Framework\+Plugin$" "https://plugins.jenkins.io/robot" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/SSH\+Slaves\+plugin$" "https://plugins.jenkins.io/ssh-slaves" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Bitbucket\+Push\+And\+Pull\+Request\+Plugin$" "https://plugins.jenkins.io/bitbucket-push-and-pull-request" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Checkstyle\+Plugin$" "https://plugins.jenkins.io/checkstyle" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Script\+Security\+Plugin$" "https://plugins.jenkins.io/script-security" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Git\+Client\+Plugin$" "https://plugins.jenkins.io/git-client" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Gradle\+Plugin$" "https://plugins.jenkins.io/gradle" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Bitbucket\+Branch\+Source\+Plugin$" "https://plugins.jenkins.io/cloudbees-bitbucket-branch-source" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Docker\+Pipeline\+Plugin$" "https://plugins.jenkins.io/docker-workflow" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/OWASP+Dependency-Check\+Plugin$" "https://plugins.jenkins.io/dependency-check-jenkins-plugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Promoted\+Builds\+Plugin$" "https://plugins.jenkins.io/promoted-builds" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Amazon\+EC2\+Plugin$" "https://plugins.jenkins.io/ec2" [NC,L,QSA,R=301]
+
+# Non plugin rewrites
+# already migrated, so don't need wiki exporter exception
+RewriteRule "^/display/JENKINS/Adopt\+a\+Plugin$" "https://jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Disable\+security$" "https://jenkins.io/doc/book/system-administration/security/#disabling-security" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Securing\+Jenkins$" "https://jenkins.io/doc/book/system-administration/security/" [NC,L,QSA,R=301]
+RewriteRule "^/display/SECURITY/Jenkins\+Security\+Advisory\+2015-11-11$" "https://jenkins.io/security/advisory/2015-11-11/" [NC,L,QSA,R=301]
```

Comment:
found this one too:

```suggestion
RewriteRule "^/display/SECURITY/Jenkins\+Security\+Advisory\+2015-11-11$" "https://jenkins.io/security/advisory/2015-11-11/" [NC,L,QSA,R=301]
RewriteRule "^/display/JENKINS/Logo$" "https://jenkins.io/artwork/" [NC,L,QSA,R=301]
```

Reply:
oh this is just a new line, sure i'll add it to the list